### PR TITLE
Adding Canada Olympic Park

### PR DESCRIPTION
**Pull request by matthewhall-ca**

### CODE REVIEW

File: lib/resorts/canada-olympic-park/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  selector: '.ancillary-flyout__status table:first-child tr:not(:first-child)',
+  parse: {
```
Comment (pirxpilot) [inline lib/resorts/canada-olympic-park/index.js:3]: I am pretty sure that `{ name: 0, status: 1 }` is liftie's default - you can probably just remove the `parse` element here

**Comment by pirxpilot** [comment]
The b91f91a does not seem right here... or anywhere else for that matter ;-) Or am I missing something?

File: lib/resorts/canada-olympic-park/index.js (hunk 1 above)
Comment (matthewhall-ca) [reply lib/resorts/canada-olympic-park/index.js]: Cool! Removed now

**Comment by matthewhall-ca** [comment]
@pirxpilot Had a brain fart and switched the wrong lat/lng 🤦 fixed now!

**Comment by pirxpilot** [comment]
Thanks. Looks good now. Do you want to squash the history or you'd rather I do that?

**Comment by matthewhall-ca** [comment]
Feel free to squash on your end if Github has an easy squash-on-merge system! I use Gitlab more in my day to day

**Comment by pirxpilot** [comment]
Thanks.
Merged as e0e6ae5
Should be live now.

**Pull request closed without merging**